_posts/2025-08-10-enhancing-security-metadata-in-bcr.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: posts
+title: Enhancing Security Metadata in BCR with Attestation Support
+authors:
+  - loosebazooka
```

Comment:
You also need to add yourself to https://github.com/bazelbuild/bazel-blog/blob/master/_config.yml

Reply:
kk done and updated the acknowledgements